packages/main/src/Card.js

```diff
@@ -166,26 +82,19 @@ const metadata = {
  * The <code>ui5-card</code> is a component that represents information in the form of a
  * tile with separate header and content areas.
  * The content area of a <code>ui5-card</code> can be arbitrary HTML content.
- * The header can be used through several properties, such as: <code>titleText</code>, <code>subtitleText</code>, <code>status</code>
- * and two slots: <code>avatar</code> and <code>action</code>.
+ * The header can be used through slot <code>header</code>. For which there is a <code>ui5-card-header</code> component to achieve the card look and fill.
  *
- * <h3>Keyboard handling</h3>
- * In case you enable <code>headerInteractive</code> property, you can press the <code>ui5-card</code> header by Space and Enter keys.
+ * Note: We recommend the usage of <code>ui5-card-header</code> for the header slot, so advantage can be taken for keyboard handling, styling and accessibility.
  *
  * <h3>CSS Shadow Parts</h3>
  *
  * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
- * <br>
- * The <code>ui5-card</code> exposes the following CSS Shadow Parts:
- * <ul>
- * <li>title - Used to style the title of the card</li>
- * <li>subtitle - Used to style the subtitle of the card</li>
- * <li>status - Used to style the status of the card</li>
- * </ul>
  *
  * <h3>ES6 Module Import</h3>
  *
  * <code>import "@ui5/webcomponents/dist/Card";</code>
+ * <br>
+ * <code>import "@ui5/webcomponents/dist/CardHeader.js";</code> (for <code>ui5-card-header</code>)
  *
  * @constructor
  * @author SAP SE
```

Comment:
add `@appenddocs CardHeader` to the jsdoc similar to [here](https://github.com/SAP/ui5-webcomponents/blob/684ccbf995b4cdd8f0db25bb952c3ebebbcd5d0e/packages/fiori/src/Timeline.js#L51).